src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/config/PropertiesConfigurationTest.kt

```diff
@@ -0,0 +1,57 @@
+package uk.gov.justice.digital.hmpps.whereabouts.config
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.context.ApplicationContext
+import org.springframework.test.context.ContextConfiguration
+import org.springframework.test.context.junit4.SpringRunner
+import java.util.*
+
+@RunWith(SpringRunner::class)
+@ContextConfiguration(classes = [PropertiesConfiguration::class])
+class PropertiesConfigurationTest {
+
+  @Autowired
+  private val context: ApplicationContext? = null
+
+  @Autowired
+  @Qualifier("whereaboutsGroups")
+  private val properties: Properties? = null
+
+//  @Autowired
+//  @Qualifier("whereaboutsEnabled")
+//  var enabled: Set<String>? = null
+
+  @Test
+  fun checkContext() {
+    assertThat(context).isNotNull
```

Comment:
better to just do `lateinit var` on `context` and `properties` and then you won't need them to be nullable